### ISSUE TITLE
hummingbird - In Checkout process, Personal Information tab, wording in incorrect

### ISSUE THREAD

**Issue by florine2623** [PrestaShop/PrestaShop#35297]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Wording is incorrect, seen with @ElodieOS.

It is now "New customer", it should be "Order as guest"

<img width="786" alt="Screenshot 2024-02-08 at 10 51 38" src="https://github.com/PrestaShop/PrestaShop/assets/16019289/3b646ef2-8d58-471c-867e-f45d22f960c8">

### Expected behavior

Should be like it was in classic theme :

<img width="735" alt="Screenshot 2024-02-08 at 10 51 43" src="https://github.com/PrestaShop/PrestaShop/assets/16019289/94abaec5-206d-4d02-8710-238d33c03a5e">

### Steps to reproduce

1. Install hummingbird theme
2. Go to FO > Add a product to cart > Proceed to checkout
3. See in Step 1 - Personal Information, wording is incorrect :x:

### PrestaShop version(s) where the bug happened

9.0.0

### PHP version(s) where the bug happened

8.1

**Comment by Hlavtox** [comment]
@ElodieOS This is an expected behavior, the new wording was implemented on purpose.

Why?

This tab serves as both **guest order** and **registration**.

**Comment by ElodieOS** [comment]
@Hlavtox this isn't a stupid thing from the classic theme, it's logical because "New customer" means you're going to create an account, but that's not the case here. You can order without creating an account (it's an option) and it's better that way. Users don't always like to create an account, even though it can make shopping easier. So it's more reassuring to understand straight away that creating an account is optional. It's a good thing for the conversion rate.

**Comment by Hlavtox** [comment]
No, new customer doesn't mean registration.

I order as a guest, I am a new customer.

**Comment by ElodieOS** [comment]
I don't agree, you may prefer to order as a guest several times on the same site, precisely because you simply don't want to create an account. 
Also, the aim of "order as guest" is to be able to order quickly without an account, not to create one. It's an option, but that's not the point.

**Comment by Hlavtox** [comment]
@ElodieOS You don't have to tell me that, I was the one who implemented the possibility to order as a guest multiple times into Prestashop. 👍

**Comment by kpodemski** [comment]
@ElodieOS 

If you decide to purchase, you'll become a customer, no matter if you create an account or not.

"Order as a guest" gives a wrong impression for some that "if I order, they won't have my data", or "If I fill in this form, they won't contact me with any emails!" but it is a wrong assumption.

On the other hand, why would you think that "New customer" means creating an account and not ordering as a guest? 🤔 

Eventually, "New customer" is good for:
- those who will buy without an account
- those who will buy and decide to create an account

because no matter what, you will become a customer :)

To me, "New customer" perfectly solves all the scenarios.

cc @PrestaShop/product-council

**Comment by SharakPL** [comment]
TBH both ways are illogical to me.
'New customer' suggests that you do create an account, when if fact you don't.
To understand what 'Order as a Guest' means customers need to know what is this Guest account (sic!).

It needs to be clear: **Order without account** or **Quick order (no account)**

**Comment by kpodemski** [comment]
@SharakPL also you suggested "Quick order (no account)", ordering without providing password is quicker but is it really worth mentioning it? is it that quicker?

and, last but not least, you have two tabs, for providing the details and to sign in, so unless you want to redesign checkout none of these options describes what happens

**Comment by MatShir** [comment]
IMO kinda agree as well with @SharakPL 

> 'New customer' suggests that you do create an account, when in fact you don't.

I think it is bc "sign in" is next to "new customer", it kinda say implicitly "can't sign" then become a "new customer" by creating one account.

**Comment by SharakPL** [comment]
Do we even need to name this form? IMO it should be simple form without tabs and panels with a single "Sign in" button on top that would open a modal or just redirect to `example.com/login?back=https//example.com/order` page.

For one of the clients I modified it like this:

![obraz](https://github.com/PrestaShop/PrestaShop/assets/5007456/ecd644a1-8b66-418a-ae3d-26153e8d2d11)
![obraz](https://github.com/PrestaShop/PrestaShop/assets/5007456/2ff63955-b2e2-4404-a7fa-dd595324aed3)

So all it would need now is a button:

![obraz](https://github.com/PrestaShop/PrestaShop/assets/5007456/a19b3ef5-a2c8-4ce7-a37c-f1e162dcca02)

**Comment by SharakPL** [comment]
> @SharakPL You are creating something new, a customer. Registered or guest, aren't you a customer?

@Hlavtox it's quite clear in Prestashop who is a customer - a registered account 😉

![obraz](https://github.com/PrestaShop/PrestaShop/assets/5007456/7f39ef8f-3247-45b3-9246-86c978059dab)

Although it's not very logical either since the definition of a customer is:

> a person who buys goods or services from a shop or business.

Key word: **buys!**
You can have an account, but no orders placed yet so you're NOT in fact a customer 🙈

**Comment by Julievrz** [comment]
"New customer" is placed next to "Sign in". For this reason, one may think that it means "Create an account". "Order as a guest" seems more logical as the option of creating an account is just that, an option. 
An alternative would be to add a tab "Create account". It would be the same form as in the "Order as a guest" tab + the password field. 
I agree that we could also remove the tabs and add the option to sign in or create an account elsewhere.

**Comment by kpodemski** [comment]
> I agree that we could also remove the tabs and add the option to sign in or create an account elsewhere.

This doesn't sound like a lot of work and would be an interesting improvement for the checkout process.

**Comment by Hlavtox** [comment]
@SharakPL Are we really going to argue that customer is not a customer because somebody named a default group in some way 10 years ago? :D

- https://en.wikipedia.org/wiki/Customer - In sales, commerce and economics, a customer (sometimes known as a client, buyer or purchaser) is the recipient of a good, service, product or an idea, obtained from a seller, vendor or supplier via a financial transaction or an exchange for money or some other valuable consideration.
- https://dictionary.cambridge.org/dictionary/english/customer - a person who buys goods or a service
- https://www.merriam-webster.com/dictionary/customer - one that purchases a commodity or service
